### ISSUE TITLE
URL Based Extraction Ignores Output Name

### ISSUE THREAD

**Issue by sroberts** [sroberts/jager#62]
So I was trying to extract [a random link](http://www.secureworks.com/cyber-threat-intelligence/threats/skeleton-key-malware-analysis/) and got yelled at for not specifying an output directory (I thought it would just output to STDOUT (which it probably should if we want to be all Unixy) and it gave me an error I didn't specify an output path with `-o`. So I specified that, then tried to cat the file I specified and got a "file not found". I looked back at the output and it gave it it's own name and ignored mine.

So basically it should not force a `-o` output (and name things itself) or allow that and follow the given name.

That said how do people feel about a more unixy approach?
